docs/runbooks/health-endpoint.md

```
@@ -0,0 +1,187 @@
+```bash
+curl -fsS http://localhost:53550/health/subsystems | jq
+```
+
+Returns `{ dormant, subsystems }` only. Same rate-limit allowlist as
+`/health` (bypasses the rate limiter).
+
+## Probe recipes
+
+### Load balancer
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)